scripts/run_integration_tests.sh

```diff
@@ -165,6 +166,11 @@ for i in "$@"; do
     shift
     ;;
 
+    --specs=*)
```

Comment:
Mmm this is one of my "personal" changes to run the tests faster that slipped into the pull request. I can revert it on next commit, basically is for running only one spec of protactor tests.